plugin/storage/grpc/factory.go

```diff
@@ -42,13 +43,14 @@ var ( // interface comformance checks
 
 // Factory implements storage.Factory and creates storage components backed by a storage plugin.
 type Factory struct {
-	metricsFactory metrics.Factory
-	logger         *zap.Logger
-	tracerProvider trace.TracerProvider
-	config         Config
-	services       *ClientPluginServices
-	remoteConn     *grpc.ClientConn
-	host           component.Host
+	metricsFactory      metrics.Factory
+	logger              *zap.Logger
+	tracerProvider      trace.TracerProvider
+	config              Config
+	services            *ClientPluginServices
+	remoteConn          *grpc.ClientConn
```

Comment:
I'd recommend more symmetrical names: tracedRemoteConn and untracedRemoteConn. Same pattern for other pairs like telset and opts.